src/ontology/mondo-edit.obo

```diff
@@ -248868,7 +248867,7 @@ xref: OMIM:605841 {source="MONDO:equivalentTo"}
 xref: Orphanet:2073 {source="OMIM:605841"}
 xref: UMLS:C1853901 {source="OMIM:605841", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline"}
 is_a: MONDO:0100049 {source="DC-OMIM:605841"} ! narcolepsy, susceptibility to
-relationship: predisposes_towards MONDO:0021107 {source="OMIM:605841"} ! narcolepsy
+relationship: predisposes_towards MONDO:0021107 {source="OMIM:605841"} ! hereditary narcolepsy
```

Comment:
I think this should be 'predispose towards" **narcolepsy**, and not "hereditary narcolepsy".

Because a variation in gene A does not predispose to a disease created by a variation in gene B

Reply:
good catch, thanks